forge-gui/res/cardsfolder/upcoming/realmbreaker_the_invasion_tree.txt

```diff
@@ -0,0 +1,14 @@
+Name:Realmbreaker, the Invasion Tree
+ManaCost:3
+Types:Legendary Artifact
+A:AB$ Mill | Cost$ 2 T | NumCards$ 3 | ValidTgts$ Opponent | TgtPrompt$ Select target opponent | SubAbility$ DBChangeZone | StackDescription$ {p:Targeted} mills three cards. {p:You} puts a land card from their graveyard onto the battlefield tapped under {p:You}'s control. It gains "If this land would leave the battlefield, exile it instead of putting it anywhere else." | SpellDescription$ Target opponent mills three cards. Put a land card from their graveyard onto the battlefield tapped under your control. It gains "If this land would leave the battlefield, exile it instead of putting it anywhere else."
```

Comment:
Break up the SpellDescription (and SpellDesc if necessary) to the appropriate lines. Otherwise your StackDesc will have redundant / weird lines.

Reply:
Fixed stack description!